### PR TITLE
fix: new sidebarv2

### PR DESCRIPTION
**Pull request by vinckr**
- **chore: update docusaurus to 3.X**
- **chore: move the message descriptions to their own file**
- **chore: format**
- **fix: compilation issues**
- **chore: formatting**
- **chore: upgrade versions in workflows**
- **chore: downgrade mermaid to 9.3**
- **chore: format**
- **chore: add sharp??**
- **chore: format**
- **chore: u**
- **fix: visual regressions**
- **chore: format**
- **chore: cleanup**
- **feat(wip): rework sidebar**
- **fix: new sidebar**
- **fix: new sidebar**
- **fix: links**
- **fix: new sidebar**
- **fix: new sidebar**
- **fix: new sidebar add oauth2**
- **fix: add keto and reorder a bit**
- **fix: new sidebar**
- **chore: update docusaurus to 3.X**
- **chore: move the message descriptions to their own file**
- **chore: format**
- **fix: compilation issues**
- **chore: formatting**
- **chore: downgrade mermaid to 9.3**
- **chore: format**
- **fix: visual regressions**
- **feat(wip): rework sidebar**
- **chore: format**
- **chore: format**
- **fix: build**
- **fix: sidebar**
- **chore: fix build**
- **chore: fix build**
- **chore: fix build**
- **chore: fix build**
- **chore: adjust the new sidebar (#1788)**
- **chore: remove licenses for now**
- **fix: ory identities overview**
- **chore: remove unneeded overviews**
- **fix: migration tutorial**
- **fix: add self-hosting and top bar items**
- **fix: merge identities intro with kratos intro**
- **fix: simply Ory CLI reference sidebar**
- **fix: simply Ory CLI reference sidebar**
- **fix: simply Ory CLI reference sidebar**
- **fix: simply Ory CLI reference sidebar**
- **fix: broken link**
- **chore: fix broken links**
- **chore: add missing docs to sidebar**
- **u**
- **u**

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by vinckr** [comment]
missing from sidebar:


- [x]  [oathkeeper/scope-strategies-hydra](https://docs-bcvkhfq3k-ory.vercel.app/docs/oathkeeper/oauth2-token-introspection-hydra) (WIP)
- [x]  _todo/proxy-jwt (empty)
- [x]  kratos/concepts/ui-messages (is embedded elsewhere)
- [x]  [debug/smtp-courier-1](https://docs-bcvkhfq3k-ory.vercel.app/docs/debug/smtp-courier-1) (debug)
- [x]  hydra/guides/common-oauth2-openid-connect-flows (skipped through redirects)

**Review by aeneasr — APPROVED** [review]
This really is, so good.

File: docs/kratos/social-signin/30_apple.mdx [hunk 1]
```diff
@@ -254,3 +254,7 @@ import SocialSigninTroubleshooting from '../_common/social-sign-in-troubleshooti
 
 Make sure to either add your apps current identifier to the `additional_id_token_audiences` field or set it as the Client ID of
 the provider in the Ory Console.
+
+```
+
+```
```
Comment (aeneasr) [inline docs/kratos/social-signin/30_apple.mdx:260]: remove

File: docs/kratos/bring-your-own-ui/01_overview.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-id: custom-ui-overview
+id: index
```
Comment (aeneasr) [inline docs/kratos/bring-your-own-ui/01_overview.mdx:2]: Unfortunately, this doesn't work (the file needs to be named index too). At the moment, this page is unavailable:

https://www.ory.sh/docs/kratos/bring-your-own-ui/index
https://www.ory.sh/docs/kratos/bring-your-own-ui

Please revert this change!

File: docs/guides/ip-allowlist.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-id: allowlist
```
Comment (aeneasr) [inline docs/guides/ip-allowlist.mdx:2]: Revert or vercel

File: docs/guides/cli/15_config-identity-service.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-id: identity-cli
```
Comment (aeneasr) [inline docs/guides/cli/15_config-identity-service.mdx:2]: Revert or add to vercel.json

File: docs/kratos/mfa/30_sms.mdx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-id: mfa-via-sms
+id: sms
```
Comment (aeneasr) [inline docs/kratos/mfa/30_sms.mdx:2]: Revert or vercel.json

File: docs/kratos/passwordless/00_overview.mdx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
-id: passwordless
+id: overview
```
Comment (aeneasr) [inline docs/kratos/passwordless/00_overview.mdx:2]: Revert or vercel.json

File: docs/kratos/passwordless/06_code.mdx [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-id: one-time-code
+id: code
```
Comment (aeneasr) [inline docs/kratos/passwordless/06_code.mdx:2]: Revert or vercel.json

also i think "email code" or "email-sms-code" would be better. or "magic code"

**Review by aeneasr — COMMENTED** [review]
@vinckr please address the comments. Sorry that I missed them during initial review. Right now, several google search results are broken, like this link:


Screenshot 2024-08-31 at 15 59 08

Screenshot 2024-08-31 at 15 59 10
